_posts/en/newsletters/2021-06-23-newsletter.md

```diff
@@ -0,0 +1,198 @@
+    BIP125 behavior.  Optech encourages any such merchants to
+    respond to the mailing list thread.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Trezor Suite adds RBF support:**
+  Trezor's wallet software, Trezor Suite, added [support for Replace-by-Fee
+  (RBF)][trezor rbf] in version 21.2.2. RBF is on by default and also supported
+  by some of Trezor's hardware devices.
+
+- **Lightning Labs announces Terminal Web:**
+  In a recent [blog post][lightning labs terminal web blog], Lightning Labs
+  describes their web-based Lightning node scoring dashboard, [Terminal
+  Web][terminal web].
+
+- **Specter v1.4.0 released:**
+  [Specter v.1.4.0][specter v1.4.0] adds a feature to ["cancel" a
+  transaction][specter 1197] using BIP125 opt-in [Replace-by-Fee (RBF)][topic
+  rbf].
+
+- **Phoenix adds LNURL-pay:**
+  ACINQ's mobile wallet [Phoenix][phoenix wallet] added support for the
+  [LNURL-pay][lnurl-pay github] protocol in its [v1.4.12 release][phoenix
+  1.4.12].
+
+- **JoinMarket v0.8.3 released:**
+  [JoinMarket v0.8.3][joinmarket v0.8.3] adds the ability to provide custom
+  change addresses and an Electrum-compatible segwit `signmessage`
+  implementation.
+
+## Preparing for taproot #1: bech32 sending support
+
+*The first segment in a weekly series about how developers and service providers
+can prepare for the upcoming activation of taproot at block height
+{{site.trb}}.*
+
+{% include specials/taproot/00-bech32m.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.13.0-beta][] is a new major release that improves feerate
+  management by making [anchor outputs][topic anchor outputs] the
+  default commitment transaction format, adds support for using a pruned
+  Bitcoin full node, allows receiving and sending payments using Atomic
+  MultiPath ([AMP][topic multipath payments]), and increases LND's
+  [PSBT][topic psbt] capabilities, among many other improvements and bug
+  fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #21365][] adds the ability for the wallet to create
+  signatures for [taproot][topic taproot] spends---both keypath spends
+  using only the P2TR public key and scriptpath spends using a
+  [tapscript][topic tapscript].  The wallet can also sign for
```

Comment:
I would have taken "tapscript" to refer to the Script version (the set of opcodes) allowed in a Tapleaf. Maybe one of these:
```suggestion
  using only the P2TR public key and scriptpath spends using
  [tapscript][topic tapscript].  The wallet can also sign for
```

```suggestion
  using only the P2TR public key and scriptpath spends using the
  [Taptree][topic Taproot].  The wallet can also sign for
```

Reply:
I'm not 100% sure, but I think BIP342 is using "tapscript" the same way I am (emphasis in original):

> The script as defined in BIP341 (i.e., the penultimate witness stack element after removing the optional annex) is called the **tapscript** and is decoded into opcodes [...]

I don't have a strong feeling here.  If you do, please say so and I'll make one of your suggested changes.

Reply:
Nope, all good. I didn't realize it was also used like this!